### ISSUE TITLE
Random augmentations aren't random?

### ISSUE THREAD

**Issue by PaulOxxx1** [dbolya/yolact#455]
Hello everyone, I hope I'm not making a rookie mistake here ... but when I try to train yolact on my custom dataset, all these "random" augmentations are only "random" once per execution. Meaning, for each distinct run of `python train.py ...` on my command line, I only get one set of random flips and rotations and so on....

Is this the intended way? I thought it's supposed to change around with every iteration / epoch ?

I'm essentially talking about all occurrences of `random.randint()` in this file.
https://github.com/dbolya/yolact/blob/b97e82d809e5e69dc628930070a44442fd23617a/utils/augmentations.py#L446

**Comment by PaulOxxx1** [comment]
I have a test dataset with three images. I have chucked in a good ol' ```print(k)``` inside the `RandomRot90` class to see all the different values of k that are "randomly" chosen. 

I then get this:

![image](https://user-images.githubusercontent.com/12044204/82822329-e6f79880-9ea5-11ea-92a8-1f341ff96f8a.png)

In another run I may get this:

![image](https://user-images.githubusercontent.com/12044204/82822436-13131980-9ea6-11ea-9094-5221c4db2065.png)

Which means I do not get different amounts of 90 degree flips in each training epoch like I should get ... right? Aren't augmentations a way to artificially increase the size of the dataset?

**Comment by abhigoku10** [comment]
@PaulOxxx1  thanks for sharing this observation, so u meant to say the random augmentations is not randomly changing in a given range for each iterations but its constant for a each training

**Comment by PaulOxxx1** [comment]
I tested this again, since I now have a slightly larger dataset I was able to turn my batch size up from one ... and it seems to be working better now. The sequence of integers still seems to be the same for all different processes running in parallel, but I guess this is something that's not easy to fix perhaps.

Even though I still don't fully trust the randomness, I think it may be sufficiently random for me to close this issue.
